packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/translations/Messages_fr_FR.json

```
@@ -0,0 +1,3 @@
+{
+  "cluster_informations_general": "Informations générales"
```

put key in english format
```suggestion
  "cluster_general_information": "Informations générales"
```